### DISCUSSION TITLE
magnetometer

### DISCUSSION THREAD

**Discussion by AliRamshini** [rodralez/NaveGo#43] [Q&A] [answered]
Hi Thank you for your best team
I want to studying for learning the magnetometer but in the first I haven't any data as you point to [imu.mb] and second I haven't your function [ myUD(S.P) ]  
please help me to learning it.
thank you so much .

**Comment by ravindraji4** [comment, marked as answer]
Hi AliRamshini, 

In NaveGo, no magnetometer is used for correcting the heading. In case  of magnetometer, it is refereed as IMU and AHRS sensor fusion algorithm this example can be found at [https://x-io.co.uk/open-source-imu-and-ahrs-algorithms/](https://x-io.co.uk/open-source-imu-and-ahrs-algorithms/)

Again, **no magnetometer** is used in NaveGo!!

**Comment by rodralez** [comment]
> Hi Thank you for your best team
> I want to studying for learning the magnetometer but in the first I haven't any data as you point to [imu.mb] 

Hello, I do not have currently magnetometer data to share. This is something I have to work on in order to implement the magnetometer heading system. Sorry.

> and second I haven't your function [ myUD(S.P) ]

This function is part of the UD filter, an error  stable version of the Kalman filter. It has not been released yet.
